src/rgw/rgw_common.h

```diff
@@ -547,14 +555,16 @@ struct RGWUserInfo
   RGWQuotaInfo bucket_quota;
   map<int, string> temp_url_keys;
   RGWQuotaInfo user_quota;
+  uint32_t type;
```

Comment:
@pritha-srivastava also need to update the RGWUserInfo json encoding and decoding